src/bindgen/ir/union.rs

```diff
@@ -35,9 +35,13 @@ pub struct Union {
 
 impl Union {
     pub fn load(item: &syn::ItemUnion, mod_cfg: Option<&Cfg>) -> Result<Union, String> {
-        if Repr::load(&item.attrs)? != Repr::C {
+        let repr = Repr::load(&item.attrs)?;
+        if repr.style != ReprStyle::C {
             return Err("Union is not marked #[repr(C)].".to_owned());
         }
+        if repr.align.is_some() {
+            return Err("Union is marked with #[repr(align(...))] or #[repr(packed)].".to_owned());
```

Comment:
It doesn't seem hard to support this, but if you don't, mind filing a follow-up issue?

Reply:
I can add support for it.

Reply:
Done.